### PR TITLE
Don't generate releasenotes of old branches/tags

### PR DESCRIPTION
**Pull request by sileht**
This change puts a redirect to the master releases notes page
on old version of the documentation.

### CODE REVIEW

File: gnocchi/gendoc.py [hunk 1]
```diff
@@ -107,13 +107,44 @@ def __getitem__(self, key):
...
 import sys
 import os
 
 srcdir = os.path.join("%s", "..", "..")
 os.chdir(srcdir)
 sys.path.insert(0, srcdir)
 
+version = sys.argv[1]
+# NOTE(sileht): We delete releasenotes from old documentation
+# only master will have it. This works only becauce the Gnocchi
```
Comment (jd) [inline gnocchi/gendoc.py]: because

File: gnocchi/gendoc.py [hunk 2]
```diff
@@ -107,13 +107,44 @@ def __getitem__(self, key):
...
+Releases notes can be found `here </releasenotes/index.html>`_
+
+.. raw:: html
+
+    <meta http-equiv="refresh" content="0; url=/releasenotes/index.html">
+
+
+\"\"\")
+
+print("*********************")
```
Comment (jd) [inline gnocchi/gendoc.py]: hm?

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 2400c9a783d7c6ef67d38498a0f0b47f68847a6e

File: doc/source/conf.py [hunk 3]
```diff
@@ -182,8 +182,10 @@
 scv_show_banner = True
 scv_banner_greatest_tag = True
 scv_priority = 'branches'
-scv_whitelist_branches = ('master', '^stable/(2\.1|2\.2|[3-9]\.)')
-scv_whitelist_tags = ("^[2-9]\.",)
+#scv_whitelist_branches = ('master', '^stable/(2\.1|2\.2|[3-9]\.)')
+scv_whitelist_branches = ('master', '^stable/4\.')
```
Comment (jd) [inline doc/source/conf.py]: I don't get this change.

**Comment by sileht** [comment]
On Wed, Jun 14, 2017 at 02:09:44AM -0700, Julien Danjou wrote:
>
>+version = sys.argv[1]
>+# NOTE(sileht): We delete releasenotes from old documentation
>+# only master will have it. This works only becauce the Gnocchi

I have removed this sentence, my last solution doesn't care about
loading order.

-- 
Mehdi Abaakouk
mail: sileht@sileht.net
irc: sileht

**Comment by sileht** [comment]
Pull-request updated, HEAD is now b13a4df5ae8de935aa289ac22eabcc4f3a2f77f9

File: doc/source/conf.py (hunk 3 above)
Comment (sileht) [inline doc/source/conf.py]: arf, testing stuff, to speed up the build

**Pull request merged**